src/Components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import paths from '@paths/paths';
```

Comment:
this is interesting. you have folder Paths, but you are importing it from paths.. (capital/ small letter).

Reply:
At file  tsconfig.json I wrote "@paths/*":["src/Paths/*"] , that's why in alias used small letter. Is it wrong?